RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate LivechatEditView to Typescript

## Proposed changes

Migrate LivechatEditView to Typescript, create 2 definitions used in LiveChatEditView


## Issue(s)	


## How to test or reproduce

- You should be an omnichannel agent
- Join a chat from Omnichannel's queue, then click on the header and click on the name
- You should be moved to another screen, then click on the `edit` button, on the header
- And the title's header should be `Edit`. So this is the screen.

## Screenshots

## Types of changes




## Checklist



## Further comments
